ui/config.yaml

```diff
@@ -0,0 +1,18 @@
+loglevel: DEBUG
+job:
+  job_spec_file: "../examples/MLCommons/job_spec.yaml"
```

Comment:
is this something the user is supposed to manually hange in the config file?

Reply:
yes ... the configuration is the input to the UI --- we will need, at a point in time to split and minimize the user inputs, for now I am trying to save time and put everything in one file.

Reply:
@eranra user should be able to provide the input path via the UI, not through a configuration file. Currently the user needs to type in the entire cli command in the configuration, which is not natural - the UI code should create the cli command and predict file configuration based on the provided input path/model path etc.

Reply:
If you look at the UI you will see "configuration viewer" this will change into a configuration editor. But for this phase, I am keeping the page just for "read-only" viewing and not allowing edits.

Reply:
![image](https://github.com/user-attachments/assets/611e940d-9642-41ef-a1e3-4fffa2a594cf)

We will need to expose to the user just a sub-set of the configuration to allow changes. The rest will need to be "read-only" or not even exposed to the user.

Reply:
Let's keep this for this first PR as-is and we will add those features in the next PR.